libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/parametersTab.tsx

```diff
@@ -14,9 +14,11 @@ export const parametersTab = (
   intl: IntlShape,
   dispatch: AppDispatch,
   {
+    isCreating,
     previousTabId,
     hasError,
   }: {
+    isCreating: boolean;
     previousTabId: string | undefined;
     hasError: boolean;
   }
```

Comment:
PLease define this as an interface and use the type everywhere, I see this is the same in other tabs